jupyterhub/notebook-images/overlays/additional/kustomization.yaml

```diff
@@ -9,3 +9,27 @@ resources:
 commonLabels:
   opendatahub.io/component: "true"
   component.opendatahub.io/name: jupyterhub
+
+
+configMapGenerator:
+- name: notebook-image-parameters
+  env: params.env
+generatorOptions:
+  disableNameSuffixHash: true
+vars:
+- name: image-sha-256-odh-minimal-notebook
+  objref:
+    kind: ConfigMap
+    name: notebook-image-parameters
+    apiVersion: v1
+  fieldref:
+    fieldpath: data.image-sha-256-odh-minimal-notebook
+- name: image-sha-256-odh-generic-data-science-notebook
+  objref:
+    kind: ConfigMap
+    name: notebook-image-parameters
+    apiVersion: v1
+  fieldref:
+    fieldpath: data.image-sha-256-odh-generic-data-science-notebook
+configurations:
+- params.yaml
```

Comment:
Since minimal and data science notebook are dynamic images we don't really need this configuration.